README.rst

```diff
@@ -195,9 +195,8 @@ Compiling with Flash Attention 2
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 TransformerEngine release v0.11.0 adds support for Flash Attention 2.0 for improved performance. It is a known issue that Flash Attention 2.0 compilation is
-resource intensive and requires a large amount of RAM (see `bug <https://github.com/Dao-AILab/flash-attention/issues/358>`_), which may lead to out of memory
-errors during the installation of TransformerEngine. To circumvent the issue, please try setting **MAX_JOBS=1** in the environment. If the errors persist, then
-proceed to install a supported version of Flash Attention 1 (v1.0.6 to v1.0.9).
+resource-intensive and requires a large amount of RAM (see `bug <https://github.com/Dao-AILab/flash-attention/issues/358>`_), which may lead to out of memory
+errors during the installation of TransformerEngine. Please try setting **MAX_JOBS=1** in the environment to circumvent the issue. If the errors persist, install a supported version of Flash Attention 1 (v1.0.6 to v1.0.9).
```

Comment:
We could also suggest using the 23.08+ container here, which has FAv2 installed already.